spec/unit/scope_file_access_spec.rb

```diff
@@ -80,6 +97,33 @@
       end
     end
 
+    describe "retrieve_file_content" do
+      let(:description) {
+        SystemDescription.load!("opensuse131-build",
+                                SystemDescriptionStore.new("spec/data/descriptions"))
+      }
+
+     it "returns the file content of a file stored in a tar ball in a sub directory" do
+       file_content = description.unmanaged_files.
+         retrieve_file_content("/usr/local/magicapp/weird-filenames/tränenüberströmt.txt")
+
+       expect(file_content).to eq("This is a file with umlauts in its name.\n")
+     end
+
+     it "returns the file content of a file stored in the files.tgz tar ball" do
+       file_content = description.unmanaged_files.retrieve_file_content("/etc/magicapp.conf")
+
+       expect(file_content).to eq("This is magicapp.conf\n")
+    end
+
+
+    it "raises an error if file is not found" do
+      expect {
+         description.unmanaged_files.retrieve_file_content("/does/not/exists")
+        }.to raise_error(Machinery::Errors::FileUtilsError)
+      end
```

Comment:
`end` at 124, 6 is not aligned with `it "raises an error if file is not found" do` at 120, 4